iris/admin/__init__.py

```diff
@@ -55,13 +56,20 @@ def actions(type):
     order_by = flask.request.args.get('order_by', 'user_id')
     ascending = flask.request.args.get('ascending', 'true')
     ascending = True if ascending == 'true' else False
+    combine_masks = flask.request.args.get('combine_masks', 'false')
+    combine_masks = True if combine_masks == 'true' else False
 
     actions = Action.query.filter_by(type=type)
     if ascending:
         actions = actions.order_by(getattr(Action, order_by)).all()
     else:
         actions = actions.order_by(getattr(Action, order_by).desc()).all()
 
+    if combine_masks:
```

Comment:
What if type isn't segmentation? I'm not sure if the UI currently allows that to happen, but it's probably safer to change this to `if combine_masks and type=="segmentation"`